_posts/2018-10-31-cockpit-181.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Cockpit 181
+author: mvo
+date: 2018-10-31
+tags: cockpit linux
+slug: cockpit-181
+category: release
+summary: ...
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 181.
+
+### Graph improvements
```

Comment:
We usually prefix this with the affected page. "System: Memory graph improvements"

Reply:
:+1:

Reply:
Done.